src/expr/src/agg/general.rs

```diff
@@ -110,6 +110,16 @@ fn count<T>(state: i64, _: T) -> i64 {
     state + 1
 }
 
+#[aggregate("bool_and(boolean) -> boolean")]
+fn bool_and(state: bool, input: bool) -> bool {
+    state && input
+}
+
+#[aggregate("bool_or(boolean) -> boolean")]
+fn bool_or(state: bool, input: bool) -> bool {
+    state || input
+}
```

Comment:
Is it necessary to have e2e tests for this trivial functions? Maybe DocSlt can indeed help.

Reply:
indeed. thanks for reminding.